svc/FetchService.js

```diff
@@ -135,4 +133,28 @@ export class FetchService {
             return null;
         }
     }
+
+    /**
+     * Does not handle nested objects
+     *  {
+     *      reportDef: {
+     *          date: '2018-08-29',
+     *          user: 'jDoe'
+     *      }
+     *  }
+     *
+     *  But it does handle arrays:
+     *   {
+     *      reportColumns: ['name', 'age', 'weight']
+     *   }
+     */
+    objectToQueryString(obj) {
```

Comment:
I think this is a great place to reach for a well-known library -  https://www.npmjs.com/package/qs appears to be one example.